client/src/main/java/io/pravega/client/segment/impl/AsyncSegmentInputStreamImpl.java

```diff
@@ -23,16 +23,17 @@
 import io.pravega.shared.protocol.netty.PravegaNodeUri;
 import io.pravega.shared.protocol.netty.WireCommands;
 import io.pravega.shared.protocol.netty.WireCommands.SegmentRead;
+import lombok.SneakyThrows;
```

Comment:
Please configure your IDE to put imports in alphabetical order to prevent unneeded extra diffs like this.